test/views/correctReturn/summary/OperatePackagingSiteOwnBrandsSummarySpec.scala

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.correctReturn.summary
+
+import base.SpecBase
+import models.LitresInBands
+import pages.correctReturn.{HowManyOperatePackagingSiteOwnBrandsPage, OperatePackagingSiteOwnBrandsPage}
+import play.twirl.api.Html
+import uk.gov.hmrc.govukfrontend.views.Aliases.Actions
+import views.summary.correctReturn.OperatePackagingSiteOwnBrandsSummary
+
+class OperatePackagingSiteOwnBrandsSummarySpec extends SpecBase {
+
+  "summaryList" - {
+    val lowLitres = 1000
+    val highLitres = 2000
+    val includeLevyRowsOptions = List(true, false)
+    includeLevyRowsOptions.foreach(includeLevyRows => {
+      s"should return correct elements when passed in with TRUE and litres provided and check answers is true and include levy rows $includeLevyRows" in {
+        val userAnswers = emptyUserAnswersForChangeActivity
```

Comment:
not required but you may want to change this to emptyUserAnswersForCorrectReturn to help avoid confusion.